app/controllers/coronavirus_form/additional_product_controller.rb

```diff
@@ -1,36 +1,30 @@
 # frozen_string_literal: true
 
 class CoronavirusForm::AdditionalProductController < ApplicationController
-  before_action :check_first_question_answered, only: :show
-
-  def show
-    render "coronavirus_form/#{PAGE}"
-  end
+  before_action :check_first_question_answered, except: :submit
```

Comment:
Why has this change from an "only" to an "except"?

Reply:
I avoided putting this into the application controller because I think pages like `/privacy` shouldn't have this check.

I've changed this to be an except because rubocop doesn't like specifying a before_action on an inherited method.

Reply:
So privacy can be excluded with `skip_before_action`,
e.g. https://github.com/alphagov/govuk-coronavirus-vulnerable-people-form/blob/master/app/controllers/coronavirus_form/privacy_controller.rb